apps/design-system-docs/src/components/link-card/index.tsx

```diff
@@ -0,0 +1,65 @@
+import useIsBrowser from '@docusaurus/useIsBrowser';
+import { Card, CardBody, CardMedia } from '@wordpress/components';
+import clsx from 'clsx';
+import { useId } from 'react';
+import defaultImage from './default.png';
+import styles from './style.module.scss';
+/**
+ * A card that links to a page.
+ */
+export const LinkCard = ( {
+	className,
+	href,
+	label,
+	description,
+	image = defaultImage,
+}: {
+	className?: string;
+	href: string;
+	label: string;
+	description?: string;
+	/**
+	 * Optional image to display in the card.
+	 * Should only be decorative, as it will be hidden from screen readers.
+	 */
+	image?: string;
+} ) => {
+	const descriptionId = useId();
+	const labelId = useId();
+
+	const isBrowser = useIsBrowser();
+
+	// Just for SSG (SEO). Can be removed when `Card` is forked and converted away from Emotion.
+	if ( ! isBrowser ) {
+		return (
+			<>
```

Comment:
I don't suppose the image is very important for the static server content?

Reply:
Yes, I'm marking the images as decorative anyway (empty alt) so it shouldn't matter that much? As long as the links are crawlable.